Some manpages are stupid, particularly those generated by pandoc
e.g. adb(1) in android-tools and contain something like the following:

.TH "" "" "" "" ""

This confuses mandoc when generating the mandb as it thinks there is
no category information, and it treats the manpage as a preformatted
manpage to be cat'ed. However, that is bad as it is not a preformatted
manpage. Since we don't have any of those, work around it by just
always treating it as a source manpage, because we can never treat
it as preformatted anyway (all our manapges are gzipped).

--- a/mandocdb.c
+++ b/mandocdb.c
@@ -1272,11 +1272,7 @@ mpages_merge(struct dba *dba, struct mparse *mp)
 			mpage->sec = mandoc_strdup(mlink->dsec);
 			mpage->arch = mandoc_strdup(mlink->arch);
 			mpage->title = mandoc_strdup(mlink->name);
-			if (meta == NULL) {
-				mpage->form = FORM_CAT;
-				parse_cat(mpage, fd);
-			} else
-				mpage->form = FORM_SRC;
+			mpage->form = FORM_SRC;
 		} else if (meta->macroset == MACROSET_MDOC)
 			parse_mdoc(mpage, meta, meta->first);
 		else
